src/components/Problems/Discussion.tsx

```diff
@@ -121,9 +122,14 @@ export const Discussion: FC<DiscussionProps> = ({problemId, invalidateSeriesQuer
 
               return (
                 <Stack sx={comment.state !== CommentState.Published ? {color: 'gray'} : {}} key={comment.id}>
-                  <Typography variant="h3" component="span">
-                    {comment.posted_by_name}
-                  </Typography>
+                  <Stack gap={1} direction="row" justifyContent={'space-between'}>
+                    <Typography variant="h3" component="span">
+                      {comment.posted_by_name}
+                    </Typography>
+                    <Typography variant="body2" fontWeight={275} textTransform="uppercase">
```

Comment:
chapem, ze fontWeight={275} uz niekde mame.. nie je to stastne, ale asi nevadi 😄

Reply:
tiez som to len z dakade zkopčil

kokrétne asi z príspevkov